doc/docs-requirements.txt

```diff
@@ -1,2 +1,5 @@
 tornado
 aiohttp
+Sphinx~=4.1
```

Comment:
What is `~=`?

Reply:
```
Mopidy-Dirble ~= 1.1        # Compatible release. Same as >= 1.1, == 1.*
```
See https://pip.pypa.io/en/stable/cli/pip_install/#example-requirements-file